packages/loading-skeleton/src/LoadingSkeleton/components/LoadingHeading/LoadingHeading.scss

```diff
@@ -0,0 +1,76 @@
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/animation";
+@import "~@kaizen/design-tokens/sass/spacing";
+@import "~@kaizen/design-tokens/sass/border";
+@import "~@kaizen/design-tokens/sass/typography";
+
+.base {
+  background-color: rgba($color-gray-500-rgb, 0.2);
+  border-radius: calc(#{$border-solid-border-radius} * 3);
+  top: calc(#{$spacing-md} * 0.175);
+}
+
+@keyframes fadeInOut {
+  0% {
+    opacity: 0.6;
+  }
+  50% {
+    opacity: 1;
+  }
+  100% {
+    opacity: 0.6;
+  }
+}
+
+.display-0 {
+  height: $typography-display-0-font-size;
+  margin-bottom: $typography-display-0-line-height;
+}
+
+.heading-1 {
+  height: $typography-heading-1-font-size;
+  margin-bottom: $typography-heading-1-line-height;
+}
+
+.heading-2 {
+  height: $typography-heading-2-font-size;
+  margin-bottom: $typography-heading-2-line-height;
```

Comment:
I think this might be wrong, see the screenshot below where I made the heading go to 2 lines.
<img width="950" alt="image" src="https://user-images.githubusercontent.com/763385/159852547-cd27c403-efe6-4723-bc1d-047c6c56256a.png">
`margin-bottom` shouldn't be the full lineheight but rather `lineHeight - fontSize = marginBottom`.
And be aware, this might work here, but `line-height` could in some cases be a "unit-less" value, for example just `1.5` (i.e. 1.5 times the font size).

Reply:
I have updated the values to be the calculated value.